task.go

```diff
@@ -0,0 +1,18 @@
+package libcarina
+
+type Input interface{}
```

Comment:
It's more of a snowball effect than anything else. If you allow yourself to have too many escape hatches, you lose automatic type checking at compile time as well as IDE integration. If the payloads are known and understood, you end up with a solid stable CLI that is easy to maintain. It does put pressure on the backend API though.